test/functional/test_framework/p2p.py

```diff
@@ -853,9 +855,11 @@ def send_blocks_and_test(self, blocks, node, *, success=True, force_send=False,
 
         reject_reason = [reject_reason] if reject_reason else []
         with node.assert_debug_log(expected_msgs=reject_reason):
+            if is_decoy:  # since decoy messages don't get sent - no need to wait for response
```

Comment:
nit:
```suggestion
            if is_decoy:  # since decoy messages are ignored by the recipient - no need to wait for response
```
(or "don't get processed" or something like that?)

Reply:
done.